### ISSUE TITLE
[Angular 10] after adding RadSideDrawer to my nativescript shared project with angular 10 i get error JS: ERROR TypeError: Cannot read property 'nativeView' of undefined

### ISSUE THREAD

**Issue by maticbezjak** [NativeScript/nativescript-angular#2196]
**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
 - CLI: 6.8.0-rc.4
 - Cross-platform modules:6.5.10
 - Android Runtime: 6.5.3
 - iOS Runtime: none
 - Plugin(s): "nativescript-ui-sidedrawer": "^9.0.0-rc.1",
 - NativeScript-Angular: 10.0.0-rc.5
 - Angular: 10.0.0
- NativeScript Schematics: 10.0.0-rc.0
 
**Describe the bug**
i generated a new project and added RadSideDrawer then i compile the app and get this error
i have the ngcc.config.js configured correctly you can see it in sample project
JS: ERROR TypeError: Cannot read property 'nativeView' of undefined
**To Reproduce**
ng new -c=@nativescript/schematics sass-project --shared --style=scss
tns plugin add nativescript-ui-sidedrawer
 add RadSideDrawer html and funtions
tns run android

**Expected behavior**

**Sample project**

[v10.zip](https://github.com/NativeScript/nativescript-angular/files/4952391/v10.zip)
after extract run npm install snd then tns run android

**Comment by Meyer-dev** [comment]
HI :) I had the same issue and it went away with new rc versions
of @nativescript/angular as well as nativescript-ui-sidedrawer and
nativescript-ui-listview. Just clean and rerun your project. Also remove
accommodations for nativescript-ui-sidedrawer in ngcc.config.js file, the
team has made it fully compatible with angular 10 and Ivy.

**Comment by iamabs2001** [comment]
Its working fine with angular 10. **CLOSE**

**Comment by NathanWalker** [comment]
@funder7 @lonemanotzi `@nativescript/schematics` **~10.0.0** has now been published alongside `@nativescript/angular` **~10.0.0**

Here's a couple examples if starting fresh (which are already documented) but here for quick reference:

A. Starting a new Angular web project and adding {N} to it afterwards:

```
ng new workspace
cd workspace
ng add @nativescript/schematics --skipAutoGeneratedComponent

// start apps:
npm run ios
npm run android 
```

B. Starting a new Angular NativeScript shared codebase from the start:

```
npm i -g @nativescript/schematics
ng new -c=@nativescript/schematics workspace --shared --style=scss
cd workspace

// start apps:
npm run ios
npm run android 

```

C. If updating existing @nativescript/schematics projects:

1. Delete webpack.config.js (the updated @nativesript/webpack dependency will regenerate a new correct one using latest webpack plugins). If you had customizations, use git change sets to compare the diff.
2. Update package.json as follows (at least in part - can use as a guide):

```
"scripts": {
  "clean": "npx rimraf hooks node_modules package-lock.json platforms",
  "ng": "ng",
  "start": "ng serve",
  "build": "ng build",
  "test": "ng test",
  "lint": "ng lint",
  "e2e": "ng e2e",
  "android": "tns run android --no-hmr",
  "ios": "tns run ios --no-hmr",
  "mobile": "tns run",
  "preview": "tns preview",
  "ngcc": "ngcc --properties es2015 module main --first-only",
  "postinstall": "npm run ngcc"
},
"dependencies": {
  "@angular/animations": "~10.0.0",
  "@angular/common": "~10.0.0",
  "@angular/compiler": "~10.0.0",
  "@angular/core": "~10.0.0",
  "@angular/forms": "~10.0.0",
  "@angular/platform-browser": "~10.0.0",
  "@angular/platform-browser-dynamic": "~10.0.0",
  "@angular/router": "~10.0.0",
  "core-js": "^2.6.9",
  "@nativescript/angular": "~10.0.0",
  "@nativescript/core": "rc",
  "@nativescript/theme": "~2.2.1",
  "reflect-metadata": "~0.1.12",
  "rxjs": "~6.5.5",
  "tslib": "1.10.0",
  "zone.js": "~0.10.2"
},
"devDependencies": {
  "@angular/cli": "~10.0.0",
  "@angular/compiler-cli": "~10.0.0",
  "@angular-devkit/build-angular": "~0.1000.2",
  "@nativescript/tslint-rules": "~0.0.5",
  "@types/jasmine": "~3.5.0",
  "@types/jasminewd2": "~2.0.3",
  "@types/node": "^12.11.1",
  "codelyzer": "^5.1.2",
  "jasmine-core": "~3.5.0",
  "jasmine-spec-reporter": "~4.2.1",
  "karma": "~4.4.1",
  "karma-chrome-launcher": "~3.1.0",
  "karma-coverage-istanbul-reporter": "~2.1.0",
  "karma-jasmine": "~3.0.1",
  "karma-jasmine-html-reporter": "^1.4.2",
  "@nativescript/webpack": "~2.0.0",
  "protractor": "~5.4.3",
  "ts-node": "~8.3.0",
  "tslint": "~6.1.0",
  "typescript": "~3.9.0",
  "node-sass": "^4.7.1"
}
```

3. Clean your projects with `npm run clean` if using the clean scripts from the example scripts above.
